scanners/ionic/ionic.go

```diff
@@ -222,8 +195,24 @@ func (scanner *Scanner) Options() (models.OptionNode, models.Warnings, models.Ic
 	}
 	// ---
 
+	// Configure Cordova
+	cordovaConfigExist, err := pathutil.IsPathExists(filepath.Join(projectRootDir, "config.xml"))
+	if err != nil {
+		return models.OptionNode{},
+			warnings,
+			nil,
+			fmt.Errorf("failed to search for config.xml file, error: %s", err)
+	}
+
+	log.TPrintf("config.xml: %s", filepath.Join(projectRootDir, "config.xml"))
+
+	if !cordovaConfigExist {
+		warning := fmt.Sprintf("Cordova config.xml not found.")
```

Comment:
Should not this be an error?

We say in this case the user should avoid using the auto configurator, but we allow them to continue. 
Other than that either if the user continues with the generated workflow or goes with manual configuration, the final workflow will contain the same steps (which are not working well with Ionic Capacitator projects).

I think we should do this:
- let the user continue with the auto-generated config (manual config would result in the same steps, just inputs need to be filled by hand)
- update the warning based on the previous and mention that after the add new app flow use workflow editor to configure your project.

Reply:
New error message:

`Our auto-configurator only supports Ionic projects with Cordova at the moment. If you’re trying to add a project with Ionic Capacitor, or something else, some Steps in your automatically generated Workflow might fail. To fix this, replace the failing Steps with script Steps in the Workflow editor later.`